aviary/docs/user_guide/subsystems.ipynb

```diff
@@ -72,7 +72,7 @@
     "\n",
     "for func in expected_flow:\n",
     "    getattr(AviaryProblem, func)\n",
-    "    # ********NOTE********\n",
+    "    # ****NOTE****\n",
     "    # check functions in each function?"
```

Comment:
@johnjasa

Reply:
Yes, I think just checking that each method exists in `SubsystemBuilderBase` is all you need to do here. Let me know if that's not your question.